srtcore/core.cpp

```diff
@@ -8294,6 +8292,16 @@ void srt::CUDT::processCtrlAck(const CPacket &ctrlpkt, const steady_clock::time_
 
             m_tsLastRspAckTime = currtime;
             m_iReXmitCount         = 1; // Reset re-transmit count since last ACK
+            if(m_iFlowWindowSize == 0)
+            {
+                m_bBufferWasFull = true;
+            }
+            else
+            {
+                m_bBufferWasFull = false;
+                m_pSndQueue->m_pSndUList->update(this, CSndUList::DONT_RESCHEDULE);
+
+            }
```

Comment:
I'm not sure about this part. At least the `m_bBufferWasFull` is the RCV side variable.
Rescheduling sending only upon full ACK should be enough.

Reply:
This was supposed to be removed, I forgot to check lite ack when cleaning the code, thanks for reviewing